cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -57,8 +58,12 @@ def check_power_options(soft_assert, instance, power_state):
     """ Checks if power options match given power state ('on', 'off')
     """
     must_be_available = {
+        AzureInstance: {
+            'on': [AzureInstance.STOP, AzureInstance.SOFT_REBOOT, AzureInstance.TERMINATE],
+            'off': [AzureInstance.START, AzureInstance.TERMINATE]
+        },
         EC2Instance: {
-            'on': [EC2Instance.STOP, EC2Instance.SOFT_REBOOT, EC2Instance.TERMINATE],
+            'on': [EC2Instance.STOP, EC2Instance.SUSPEND, EC2Instance.TERMINATE],
```

Comment:
All right, that sounds valid but you are editing expected EC2Instance states here, not Azure.